### PR TITLE
added back to login button

### LINKED ISSUE #3011: On the state affiliation selection page, give users an option to exit
In the log in refactor ticket #2706, we determined that once folks get to where they choose their state affiliation, they are effectively logged in to the system and need a way to exit this screen. 

### Possible solutions 
We could do one of two things - 
~Show the regular header at this point and provide a log out~
We decided on. - * Match the "Back to Login" link that we're using in other parts of the system

Example - 

### This task is done when...
 - [x] a decision has been made on which direction to go in
 - [ ] Decision is: add the Back to log in link on the State Affiliation page
 - [x] this direction is approved by product

### PR DESCRIPTION
**Pull request by thetif**
resolves #3011 

### This pull request changes...

- adds the Back to login button to the State selection page

### This is how to verify this change...
1. Sign in using norole
2. See the Back to login button 
3. Click the Back to login button
4. The user should be logged out and returned to the login page

### This pull request is ready to merge when...

- [x] Develop has updated automated tests (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [ ] QA has manually tested and created bugs for any issues found (beyond minor fixes)
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by tbolt — APPROVED** [review]

**Comment by eStar386** [comment]
Per #3124, I am using the [regression new user test cases] (https://docs.google.com/spreadsheets/d/1dozD8rJWpIxdC1cbzRULKA9iIHy_xFjlXoNwkgu7OGs/edit?usp=sharing) to confirm the functionality. I will update this test script to reflect the new functionality created.  Please note that this already contains a section for test notes but I make sure to fill it in so you can verify how I test.

**Comment by eStar386** [comment]
During testing, I am noticed that my self created users having noticeable lag when using the back to login button vs the  norolemfa which doesn't have any lag. The next steps are to create a new user from https://test.reg.idm.cms.gov/registration.html?appid=eapd, this is for #3124.

**Comment by thetif** [comment]
Please let me know if that does fix the lag. I can't think of a reason that there would be a different speed for different users.

**Comment by eStar386** [comment]
@thetif 
Role Affected: Personal User Setup a new user in the test.reg.idm.cms.gov/registration.html?appid=eapd

Precondition: Using the link about create a new user. 
Browser: Chrome (Private Browser) 

Detailed Description of the BUG: The back to login button causes the user to be stuck on the validating your session page.

Steps To Reproduce: 
1.	Using the create user from the precondition, login. 
2.	Verify your identity 
3.	On the state affiliation selection page, select back to login button.  On the backend The new user expect for norolemfa triggers an OAuthError. 

Expected Result: Return to the Login Page

Actual Result: Stuck on the validating the page. The user must close the browser to force it close. 

https://user-images.githubusercontent.com/71447081/118310078-2145b580-b4bc-11eb-9696-1dfbd177193b.mov
Uploading Chrome Recording 2021-05-14 at 11.09.17 AM.mov…

**Comment by eStar386** [comment]
Accessible Issue
 Failures on the State Selection Page: 
Wave

File: web/src/components/AuthenticationForm.js [hunk 1]
```diff
@@ -24,6 +25,11 @@ const AuthenticationForm = ({
     if (canSubmit && onSave) onSave(e);
   };
 
+  const handleCancel = e => {
+    e.preventDefault();
+    if (cancelable && onCancel) onCancel(e);
```
Comment (tbolt) [inline web/src/components/AuthenticationForm.js:30]: fancy!

**Review by tbolt — APPROVED** [review]

**Comment by thetif** [comment]
hi @eStar386 I'm not seeing the wave issues now 

Screen Shot 2021-05-18 at 5 46 53 PM

**Review by beparticular — APPROVED** [review]

**Review by NAretakis — APPROVED** [review]
Good to go!

**Pull request merged**